src/components/layout/nav.js

```diff
@@ -1114,6 +1137,94 @@ export const NavPartners = ({ no_login_signup }) => {
     )
 }
 
+// Note: When using layout component for security page, please add type='security' and padding_top='10rem'
+export const NavSecurity = () => {
+    const nav_ref = useRef(null)
+    const button_ref = useRef(null)
```

Comment:
Are these refs being used? If not, we can remove them.

Reply:
the nav_ref is not, but the button_ref is being used...though not in the most useful way for this component 😅